about.qmd

```diff
@@ -65,6 +65,8 @@ If you would like to see a blog post on a particular topic, submit an issue on o
 - [ ] (For fun) Spatial transcriptomics plots in stained glass
 - [X] [Visualization of cellular neighborhood in gallery mode](posts/visualize-cellular-neighborhood-in-gallery-mode/index.qmd)
 - [X] [Evaluating cell segmentation error based on transcripional spatial profiles](posts/segmentation-error-evaluation/index.qmd)
+- [X] [Perturbation analysis of disease vs. control cellular neighborhoods](https://nanostring-biostats.github.io/posts/insitudiff/index.qmd)
```

Comment:
Remove `https://nanostring-biostats.github.io/` here and on the next one. Others start from `post`